src/main/java/de/retest/web/ScreenshotProvider.java

```diff
@@ -47,11 +50,15 @@ private static BufferedImage shootFullPage( final WebDriver driver ) {
 		final BufferedImage image = Shutterbug
 				.shootPage( driver, ScrollStrategy.WHOLE_PAGE, SCROLL_TIMEOUT_MS, USE_DEVICE_PIXEL_RATIO ).getImage();
 		return driver instanceof ChromeDriver
-				? resizeImage( image, image.getWidth() / SCALE, image.getHeight() / SCALE ) : image;
+				? ImageUtils.resizeImage( image, image.getWidth() / SCALE, image.getHeight() / SCALE ) : image;
 	}
 
 	private static BufferedImage shootViewportOnly( final WebDriver driver ) {
 		return Shutterbug.shootPage( driver, USE_DEVICE_PIXEL_RATIO ).getImage();
 	}
 
+	private static BufferedImage resizeImage( final BufferedImage image ) {
+		final double height = image.getHeight() * (WANTED_WIDTH / image.getWidth());
+		return ImageUtils.resizeImage( image, WANTED_WIDTH, (int) height );
```

Comment:
Have measured how expensive this is? Recently we've been quite critical when it comes to screenshots and performance. While resizing reduces the file size, i.e., less local storage needed, faster upload to rehub, … but how does this affect the actual performance of recheck-web?

Reply:
It's a one-time operation with many-time benefits. If it's too slow, user should disable screenshots.